### PR TITLE
iwyu

### PR DESCRIPTION
**Pull request by petersteneteg**
Thank you for contributing to the Inviwo repository! Great code makes great apps so please ensure the following:

**Always**
- [ ] Code is error and warning free
- [ ] Code follows the [Inviwo coding guidelines](https://github.com/inviwo/inviwo/wiki/Coding-Conventions)

**New features**
- [ ] Code / Processors follows the [Inviwo documentation guidelines](https://github.com/inviwo/inviwo/wiki/Documentation-guide)
- [ ] Code / processors are tested in [unittest and/or regresion tests](https://github.com/inviwo/inviwo/wiki/Manual#testing)

**Updated feature / Fixes** 
- [ ] Relevant documentation has been updated

**What evineroment has the code been compiled and tested on?** 
- Operating system
- Compiler/IDE
- QT/CMake/Python versions

Are you are unsure how to address these points? Please make a note here so that we can help out:

### CODE REVIEW

**Review by ResearchDaniel — DISMISSED** [review]
No idea how to use it but the results look ok.
Could you add a ReadMe.md file in the folder where you explain that it is IWYU stands for include what you use and how to use it?

File: tools/iwyu/glmimp.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+from pathlib import Path
+
+
+def begins_with(alist, prefix):
+    if len(prefix) > len(alist):
+        return False
+    return alist[:len(prefix)] == prefix
+
+
+glm_base = "/Users/peter/Work/Inviwo/inviwo/ext/glm/glm"
```
Comment (JochenJankowai) [inline tools/iwyu/glmimp.py]: Is this a generated file? Otherwise, this path should probably be relative.

**Comment by JochenJankowai** [comment]
I am not too familiar with Jenkins' capabilities but isn't this something that could run there?

File: tools/iwyu/glmimp.py (hunk 1 above)
Comment (petersteneteg) [reply tools/iwyu/glmimp.py]: Yes, but you only have to do it ones kind of, or of glm does some large changes...

**Comment by petersteneteg** [comment]
Yes, I would like to run it on Jenkins or GitHub Actions at some point... But at the moment it changes quite a lot of stuff so the interaction of it has to be done with a bit of care to avoid breaking way to much code...

File: modules/basegl/src/processors/multichannelraycaster.cpp [hunk 2]
```diff
@@ -32,9 +32,11 @@
 #include <modules/opengl/volume/volumegl.h>
 #include <modules/opengl/image/layergl.h>
 #include <modules/opengl/texture/textureunit.h>
+
```
Comment (martinfalk) [inline modules/basegl/src/processors/multichannelraycaster.cpp:31]: empty lines?

**Review by martinfalk — DISMISSED** [review]
what about qt6_x header files?

**Pull request closed without merging**